### PR TITLE
Make statement field in corporate endorsement not required

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
ECOM-6544

@schenedx @rlucioni @umar-nawaz please review

### CODE REVIEW

**Comment by umar-nawaz** [comment]
@ahsan-ul-haq These Travis tests are also failing on my PR: https://travis-ci.org/edx/course-discovery/builds/192293317

**Comment by ahsan-ul-haq** [comment]
@umar-nawaz It seems like some sort of translation issue

**Comment by rlucioni** [comment]
@umar-nawaz @ahsan-ul-haq keep an eye on https://github.com/edx/course-discovery/pull/568 for status on a fix for the build issue.

**Review by schenedx — CHANGES_REQUESTED** [review]
Can you please add a new test case to ensure we can insert null into the `statement` field?

**Comment by ahsan-ul-haq** [comment]
@schenedx added test and migration please take a look

File: course_discovery/apps/course_metadata/tests/test_models.py [hunk 1]
```diff
@@ -596,6 +596,14 @@ def setUp(self):
     def test_str(self):
         self.assertEqual(str(self.individual_endorsements), self.corporation_name)
 
+    def test_null_statement(self):
```
Comment (clintonb) [inline course_discovery/apps/course_metadata/tests/test_models.py]: This is essentially testing Django. This may be one of the few times where I say you don't need tests.

File: course_discovery/apps/course_metadata/tests/test_models.py [hunk 2]
```diff
@@ -596,6 +596,14 @@ def setUp(self):
...
+        endorsement = CorporateEndorsement.objects.create(
```
Comment (clintonb) [inline course_discovery/apps/course_metadata/tests/test_models.py]: In the future, use a factory.

```python
endorsement = CorporateEndorsementFactory(statement=None)
self.assertIsNone(endorsement.statement)
```

**Review by schenedx — APPROVED** [review]
@ahsan-ul-haq The test case came out not as I expected. I was wrong to call for a test case. I agree with @clintonb. Go ahead and remove the test case and you should be 👍
